openedx/features/course_experience/views/course_home.py

```diff
@@ -88,40 +88,56 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         Renders the course's home page as a fragment.
         """
         course_key = CourseKey.from_string(course_id)
+        course = get_course_with_access(request.user, 'load', course_key)
 
-        # Render the outline as a fragment
-        outline_fragment = CourseOutlineFragmentView().render_to_fragment(request, course_id=course_id, **kwargs)
+        # Render the course dates as a fragment
+        dates_fragment = CourseDatesFragmentView().render_to_fragment(request, course_id=course_id, **kwargs)
 
-        # Get resume course information
-        has_visited_course, resume_course_url = self._get_resume_course_info(request, course_id)
+        # Determine if the user is enrolled in the course (or is staff)
+        is_enrolled_or_staff = CourseEnrollment.is_enrolled(request.user, course_key)
```

Comment:
I'm confused.  Do we want this to be `is_enrolled_or_staff` or `is_enrolled`?  Does `is_enrolled()` return True for staff regardless?  Note, depending on whether this is a defect, or just need clearer comments, we might be missing a test that would catch this case.